stage23/protos/stivale.c

```diff
@@ -174,6 +174,12 @@ void stivale_load(char *config, char *cmdline) {
         print("stivale: Requested stack at: %X\n", stivale_hdr.stack);
     }
 
+    // Check if the requested stack is correctly aligned at a word
+    // boundary (8 bytes on x64 and 4 bytes on x86).
+    if (stivale_hdr.stack % sizeof(size_t) != 0) {
```

Comment:
This modulo could be changed into an and (which is a lot faster).

Reply:
Fixed!